wdl2cwl/main.py

```diff
@@ -275,7 +332,8 @@ def get_literal_name(
         # the literal value is what's needed
         if isinstance(expr.parent, WDL.Expr.Apply):  # type: ignore
             return expr.literal.value  # type: ignore
-        raise Exception(f"The parent expression for {expr} is not WDL.Expr.Apply")
+        parent_name = expr.parent.name  # type: ignore
+        return self.get_input(parent_name)
```

Comment:
These two lines have no test coverage. Maybe you should save them for another pull request?

Reply:
It's covered in the new file. It's needed for top_hit_id